### DISCUSSION TITLE
on off buttons for frida il2cpp

### DISCUSSION THREAD

**Discussion by VAROME** [vfsfitvnm/frida-il2cpp-bridge#379] [Q&A]
i know maars https://github.com/maarsalien/frida-android-mod-menu
but this is not working for android less than 13 , and will crash and the menu wont even show if even android 12 
but i need either it is fixed for android 10 or 11 
or another way to get an on / off buttons for the methods. .

**Comment by commonuserlol** [comment]
its may look like ad, but you can try [my project](https://github.com/commonuserlol/frida-java-menu), should work in ~a6-13 range

**Reply by VAROME** [reply]
i really hope you create a video about starting it in a new empty folder, [with VS code if you want ] cause your environment already had dealt with issues that will and are facing who just trying to do it , you can even use it with gadget , or whatever . 
i am trying now, on empty sutff, but so far the menu aint showing yet. 
so far error 
```
Error: could not parse 'E:\frida\GUII\index.js' line 1: expecting '('
    at <anonymous> (/frida/repl-2.js:1)
```
and if possible to email me a working copy  ``pccreator2official@gmail.com``

**Reply by commonuserlol** [reply]
I think you should write this to my issues, but which compiler you used? frida compile? try esbuild

**Reply by VAROME** [reply]
i have submitted the full issue with codes and everything 
i use  `esbuild` in normal vscode ide .
